django/chat/responses.py

```diff
@@ -525,6 +529,7 @@ async def add_files_to_library():
             responses = [
                 synthesizer.synthesize(query=input, nodes=sources).response_gen
                 for sources in source_groups
+                if not cache.get(f"stop_response_{response_message.id}", False)
```

Comment:
Doesn't break anything to include this, but does require ninja skills to reduce costs